dagfactory/dagbuilder.py

```diff
@@ -140,6 +140,10 @@ def get_dag_params(self) -> Dict[str, Any]:
             "on_skipped_callback",  # Not applicable at the DAG-level
             "sla_miss_callback",  # Not applicable at the default_args level
         ]:
+            if callback_type == "sla_miss_callback" and version.parse(AIRFLOW_VERSION) >= version.parse("3.1.0"):
+                # sla_miss_callbacks are fully-deprecated as of 3.1.0. No need to raise a warning, we can silently skip
```

Comment:
We should log a  warning here, instead of silently passing

Reply:
Yeah, logging may be a good idea here

Reply:
We will address this in a separate PR